pantheon-sessions.php

```diff
@@ -261,6 +261,50 @@ public static function force_first_load() {
 			}
 		}
 	}
+
+	/**
+	 * Checks whether primary keys were set and notifies users if not.
+	 */
+	public static function check_native_primary_keys() {
```

Comment:
Consider if there's a way to break this method up to something that can/should be functionally tested (It's possible the answer is no).